plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx

```diff
@@ -77,18 +80,40 @@ export const FeedbackResponseDialog = (props: FeedbackResponseDialogProps) => {
     feedbackDialogTitle = 'Please provide feedback on what can be improved',
     open,
     onClose,
+    enableValidation = true,
+    showCommentsTextBox = true,
   } = props;
   const classes = useStyles();
   const errorApi = useApi(errorApiRef);
   const feedbackApi = useApi(entityFeedbackApiRef);
-  const [responseSelections, setResponseSelections] = useState(
-    Object.fromEntries(feedbackDialogResponses.map(r => [r.id, false])),
-  );
+  const [responseSelections, setResponseSelections] = useState(() => {
+    const initialSelections = Object.fromEntries(
+      feedbackDialogResponses.map(r => [r.id, false]),
+    );
+    if (!initialSelections.hasOwnProperty('other')) {
+      initialSelections.other = false;
```

Comment:
This will always be `false` since it won't appear in the UI if the consumer of this dialog chooses not to include an `other` response